tests/chalicelib/model_transfer.py

```diff
@@ -0,0 +1,45 @@
+import datetime as dt
+from typing import Dict
+
+from cuenca_validations.types import StrictTransferRequest, TransactionStatus
+from mongoengine import DateTimeField, Document, IntField, StringField
+
+from agave.models.base import BaseModel
+from agave.models.helpers import EnumField, uuid_field
+
+
+class Transfer(BaseModel, Document):
```

Comment:
No necesitas un modelo complejo como este para hacer las pruebas. Utiliza uno más sencillo que sea útil para probar todos los casos. Las pruebas serían mucho más sencillas

Reply:
changed to more clear